presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationPushDown.java

```diff
@@ -120,20 +119,17 @@ public void testOffsetPushDown() throws ExecutionException, InterruptedException
     {
         createMessages(topicNameOffset);
         DistributedQueryRunner queryRunner = getDistributedQueryRunner();
-        String sql = format("SELECT count(*) FROM default.%s WHERE _partition_offset between 2 and 10",
-                topicNameOffset);
+        String sql = format("SELECT count(*) FROM default.%s WHERE _partition_offset between 2 and 10", topicNameOffset);
 
         ResultWithQueryId<MaterializedResult> queryResult = queryRunner.executeWithQueryId(getSession(), sql);
         assertEquals(getQueryInfo(queryRunner, queryResult).getQueryStats().getProcessedInputPositions(), 18);
 
-        sql = format("SELECT count(*) FROM default.%s WHERE _partition_offset > 2 and _partition_offset < 10",
-                topicNameOffset);
+        sql = format("SELECT count(*) FROM default.%s WHERE _partition_offset > 2 and _partition_offset < 10", topicNameOffset);
```

Comment:
nit: it would be better to inline the expression rather than to reuse the `sql` variable

Reply:
Then I would need to also inline `queryResult` and then test becomes unreadable